src/components/node-list/node-list.test.js

```diff
@@ -204,24 +204,38 @@ describe('NodeList', () => {
     it('selecting tags enables only elements with given tags', () => {
       const wrapper = setup.mount(<NodeList />);
 
-      changeRows(wrapper, ['huge'], true);
-      expect(elementsEnabled(wrapper)).toEqual([['whale', true]]);
+      changeRows(wrapper, ['Small'], true);
+      expect(elementsEnabled(wrapper)).toEqual([
+        ['salmon', true],
```

Comment:
Where these results (and all the others) manually verified to be correct e.g. using the browser?

Reply:
good point Liam - will check these specific cases on the browser again and flag them if I spot any edge cases.